package.json

```diff
@@ -34,6 +38,8 @@
     "yargs": "^3.2.1"
   },
   "devDependencies": {
+    "eslint": "^8.47.0",
+    "prettier": "^3.0.2",
```

Comment:
You may find issues with Prettier, it is well documented (even on their own site) that it conflicts with basically everything. There isn't anything it does that ESLint can't also do. I tend to avoid it so I don't force others into dealing with Prettier causing issues in their editor.